projects/ngx-location-viewer/src/lib/classes/location-viewer-map.ts

```diff
@@ -74,9 +74,16 @@ export class LocationViewerMap extends LeafletMap {
         // style is used to style lines and polygons
         featureLayerOptions.style = (feature) => {
           if (layer.colors && layer.colors.length > 0) {
-            const colorValue = feature.properties[layer.styleField];
-            const colorItem = layer.colors.find((x) => x.value === colorValue);
-            return colorItem ? colorItem : layer.colors[0];
+            let colorItem = layer.colors[0];
+            
+            // check if more coloritems are avaiable to check for custom style by value
+            if (layer.colors.length > 1)
+            {
+              const colorValue = feature.properties[layer.styleField];
```

Comment:
wat als iemand meer dan 1 layer color meegeeft maar geen stylefield gezet heeft?

Reply:
het stylefield wordt ingesteld in de GIS laag,
layer colors komen vanuit GIS
is dus conditionele styling vanuit GIS
vanuit de settings is het enkel mogelijk om deze te overrulen met 1 config